frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -666,6 +669,49 @@ export default function SurveyEdit(): JSX.Element {
                                                                 />
                                                             </div>
                                                         </LemonField.Pure>
+                                                        <LemonField.Pure
+                                                            label="Device Types"
+                                                            info="Add the device types to show the survey on. Supported values: 'Desktop', 'Mobile', 'Tablet'. Requires at least version 1.214 of posthog-js"
```

Comment:
the docs mention a few more supported values, such as Desktop, Mobile, Tablet, Console, and Wearable
You can either link to the docs ( I mentioned the PR before ) or we do an exhaustive list here

Reply:
I think linking to the docs is better because it explains the caveats with $device_type from person props as well

Reply:
do you mean the [new docs you added here](https://github.com/PostHog/posthog.com/pull/10529/files)?

Reply:
Yes

Reply:
okay, just updated it with a link to it!